sources/_subprojects_outputs/tokens/balances.yml

```diff
@@ -39,4 +39,7 @@ sources:
   - name: tokens_scroll
     tables:
       - name: balances_daily_agg
-      - name: balances_daily_agg_base
\ No newline at end of file
+      - name: balances_daily_agg_base
+  - name: tokens_optimism
+    tables:
+      - name: balances_daily
```

Comment:
`tokens_optimism` schema is already listed in line 7, please take advantage and list tables there instead

Reply:
Okay.

Reply:
will leave this unresolved until next commit with change